utils/cliutils/commandsflags.go

```diff
@@ -1815,6 +1816,9 @@ var commandFlags = map[string][]string{
 		url, user, password, accessToken, sshPassphrase, sshKeyPath, serverId, refs, glcRepo, glcDryRun,
 		glcQuiet, InsecureTls, retries, retryWaitTime,
 	},
+	CocoapodsConfig: {
+		global, serverIdResolve, serverIdDeploy, repoResolveReleases, repoResolveSnapshots, repoDeploySnapshots, repoDeployReleases,
```

Comment:
`serverIdDeploy` shouldn't be added for package managers that do not deploy packages to Artifactory.
In addition, the following options should be removed (they are all maven specific):
* repoResolveReleases
* repoResolveSnapshots
* repoDeploySnapshots

The only options that are needed are the below (see `NugetConfig` above as a reference):
* global
* serverIdResolve
* repoResolve

Reply:
Fixed